templates/ntp.conf.j2

```diff
@@ -30,6 +30,9 @@ pool {{ item }}
 # permit the source to query or modify the service on this system.
 restrict default nomodify notrap nopeer noquery
 
+# Allow pool associations
+restrict source nomodify notrap noquery
```

Comment:
Is this addition something that could make NTP's behavior different for existing role users who upgrade them suddenly have this new line in their config?

Reply:
It will make the pool associations work again. see [the bug description](https://bugs.ntp.org/show_bug.cgi?id=2657) for further information.

Reply:
fixes #85

Reply:
The error is that `nopeer` prevents using servers from a pool directive. The `man 5 ntp.conf` reads:

> **restrict** ...
> ...
>          **nopeer** ... It also includes pool associations, so if you want to use servers from a pool directive and also want to use nopeer by default, you'll want a **restrict source ...** line as well that does not include the nopeer directive.

So this PR fixes the problem with `ntpd >=4.2.7` exactly as it described in the manual.